pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -52,9 +50,9 @@ def work(self):
                                 individual_defense = pokemon_data.get("individual_defense", 0)
 
                                 iv_display = '{}/{}/{}'.format(
-                                    individual_stamina,
                                     individual_attack,
-                                    individual_defense
+                                    individual_defense,
```

Comment:
The reason that the renaming worker uses ADS format is because that is also used in the web UI. We're using this as an opportunity to make the pokemon catch worker output consistent with this format as well.